openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/director_folder/start_director.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
It will be better if the sh-file is executable.

Reply:
Done as well.

Reply:
Done as well.